src/Screens/OnceAdayDose/OnceAdayDose.Screen.tsx

```diff
@@ -139,15 +149,23 @@ const OnceAdayDose: FC = () => {
 
       // Add the new data to the copied array
       updatedStoredList.push(data);
+
       await localSchedule(updatedStoredList, 'day', medicineLocalId);
+
       await createMedicineData(updatedStoredList, accessToken);
+
+      await INSTRUCTION_MUTATION(updatedInstructionList, accessToken, medicineLocalId);
```

Comment:
add comments for functions. it will help you or other developers to know about its functionalities easily.